scenes/props/fixed_size_label/fixed_size_label.gd

```diff
@@ -17,6 +17,15 @@ extends Control
 @onready var label_container: PanelContainer = %LabelContainer
 
 
+func _set(property: StringName, value: Variant) -> bool:
+	if not is_node_ready():
+		return false
+	if property == "visible":
+		label_container.visible = value
+		return true
+	return false
+
+
```

Comment:
What about making the visibility of the label depend on the visibility of this node?, like, in _process:

```
label_container.visible = visible
```

I think it's simpler and it should work the same

Reply:
Yes, we have assignments like that in the `_process` functions everywhere. But I'm not sure, maybe this solution is better in this case? @wjt @pablitar thoughts?

Reply:
@JuanFdS I just realized that setting the label visibility in `_process` won't work for player fighting mode, because it disables processing of the PlayerInteraction node (which includes a fixed label instance). I just added a separate commit to hide the label when fighting.